docker-entrypoint.sh

```diff
@@ -20,4 +20,8 @@ if [[ -d "/opt/blueprints/nginx" ]]; then
     echo "Volume mount exists copying files"
     cp -r shelf-layouts/ nginx/
     cp -r external-frames/ nginx/
-fi
\ No newline at end of file
+fi
+
+cd shelf-layouts
+
+./upload.sh
```

Comment:
3 things:

1. This can be condensed to `./shelf-layouts/upload.sh`. This will make it easier and less error prone if we add other steps to the Dockerfile in the future.
2. Is `shelf-layouts` representative? Or would `layouts` be a better fit?
3. I would prefer if we have a `scripts` folder at the root of the repo, where we could put these things. It will make it easier to manage. My suggestion is to move the `upload.sh` script to `/scripts/upload-layouts.sh`.

Reply:
1. It can not be yet. The upload script would need some changes to not be dependent on the cwd. I couldn't find a good and simple solution to that problem so I went with `cd`.
2. Sure, `layout` is better
3. We do have that folder, but I need a solution to 1. for this to be possible

Reply:
I addressed all 3 :)

Reply:
And it occurred to me that I could have written the script in JS, because we do have Node in the container🤦‍♂️

Reply:
That can be an enhancement later on 😄